packages/service-library/src/servicelib/fastapi/celery/app_server.py

```diff
@@ -18,8 +12,8 @@ def __init__(self, app: FastAPI):
     async def on_startup(self) -> None:
         self._lifespan_manager = LifespanManager(
             self.app,
-            startup_timeout=_STARTUP_TIMEOUT,
-            shutdown_timeout=_SHUTDOWN_TIMEOUT,
+            startup_timeout=None,  # waits for full app initialization (DB migrations, etc.)
+            shutdown_timeout=None,
```

Comment:
why the shutdown timeout is set to None? is this not going to generate problems?